addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/BodyGenerator.java

```diff
@@ -100,8 +102,23 @@ public String generate(Schema<?> schema) {
     }
 
     private String generateFromArraySchema(ArraySchema schema) {
+        if (schema.getExample() instanceof String) {
+            return (String) schema.getExample();
+        }
+
         StringBuilder json = new StringBuilder();
+
+        if (schema.getExample() instanceof Iterable) {
```

Comment:
This should be done before instantiating the above `StringBuilder`.

Reply:
I've now remove the StringBuilder completely (as suggested by IntelliJ) as it unnecessary. (413785c69aa75867855b05ea62a4cb49fec98cae)